docs/demos/coast-to-coast-q4-2023.md

```diff
@@ -0,0 +1,793 @@
+
+Create a new directory for the project and `cd` into it:
+```sh
+mkdir coastal && cd coastal
+```
+
+Initialize the project:
+```sh
+operator-sdk init --domain coastal.operatorframework.io
+```
+
+Create a new API:
+```sh
+operator-sdk create api \
+    --group coastal.operatorframework.io \
+    --version v1alpha1 \
+    --kind Coast \
+    --resource --controller
+```
+
+### Build the Operator Controller and Bundle Images
+For this demo we are going to build a couple different versions of this operator:
+- `v1.0.0-alpha1`
+- `v1.0.0`
+- `v1.0.1`
+- `v1.1.0`
+- `v2.0.0`
+
+Initial setup:
+- Create the `manifests/` directory for building the plain bundle
+```sh
+  mkdir -p manifests
+```
+
+- Create a Dockerfile for the plain bundle image
+```sh
+cat << EOF > plainbundle.Dockerfile
+FROM scratch
+ADD manifests /manifests
+EOF
+```
+
+For each of the versions above, do the following:
+- Generate the manifests
+```sh
+  make generate manifests
+```
+
+- Build the controller image
+```sh
+  make docker-build IMG="quay.io/operator-framework/coastal:{VERSION}"
+```
+
+- Populate the `manifests/` directory for building the plain bundle
+```sh
+  kustomize build config/default > manifests/manifests.yaml
+```
+
+- Build the plain bundle image
+```sh
+docker build -t quay.io/operator-framework/coastal-bundle:{VERSION} -f plainbundle.Dockerfile .
+```
+
+- Push the controller image and plain bundle image
+```sh
+docker push quay.io/operator-framework/coastal:{VERSION} && \
+docker push quay.io/operator-framework/coastal-bundle:{VERSION}
+```
+
+A handy little script to run all these steps for you:
+```sh
+#!/bin/bash
+set -e
+
+mkdir -p manifests
+
+cat << EOF > plainbundle.Dockerfile
+FROM scratch
+ADD manifests /manifests
+EOF
+
+versions=( v1.0.0-alpha1 v1.0.0 v1.0.1 v1.1.0 v2.0.0 )
+for version in "${versions[@]}"
+do
+  make generate manifests
+  make docker-build IMG="quay.io/operator-framework/coastal:${version}"
+  mkdir -p manifests
+  kustomize build config/default > manifests/manifests.yaml
+  docker build -t "quay.io/operator-framework/coastal-bundle:${version}" -f plainbundle.Dockerfile .
+  docker push "quay.io/operator-framework/coastal:${version}"
+  docker push "quay.io/operator-framework/coastal-bundle:${version}"
+done
+```
+
+### Create an initial File-Based Catalog (FBC) Image
+Now we need to build and push an initial File-Based Catalog (FBC) image containing
+our bundles. To help highlight the new polling functionality, we are only going to add the `v1.0.0-alpha1`
+bundle to our catalog image in this step and as we go through the rest of the sections we will progressively
+add more of our bundles to the catalog.
+
+- Create a `catalog/` directory
+```sh
+mkdir catalog
+```
+
+- Create a Dockerfile for the FBC image
+```sh
+cat << EOF > catalog.Dockerfile
+FROM scratch
+ADD catalog /configs
+LABEL operators.operatorframework.io.index.configs.v1=/configs
+EOF
+```
+
+- Create the FBC YAML file
+```sh
+cat << EOF > catalog/index.yaml
```

Comment:
```suggestion
- Create the catalog YAML file
```sh
cat << EOF > catalog/index.yaml
```

(Also, this says YAML, but the sample provided is JSON.)

Reply:
Whoops... should be resolved in [9cecd82](https://github.com/operator-framework/operator-controller/pull/533/commits/9cecd82c89aa82af486c15b987cc97833517ffbf)